netbout-client/src/main/java/com/netbout/client/RtInbox.java

```diff
@@ -119,4 +119,9 @@ public Bout bout(final long number) {
             }
         };
     }
+
+    @Override
+    public Iterable<Bout> search(final String term) throws IOException {
+        throw new UnsupportedOperationException("#search()");
```

Comment:
@komaz maybe for now we can just return empty iterable, there is no need to react so strongly about missing implementation.